sentry/src/main/java/io/sentry/Sentry.java

```diff
@@ -23,6 +25,71 @@ private Sentry() {
     }
 
     /**
+     * Initialize and statically store a {@link SentryClient} by looking up
+     * a {@link Dsn} and automatically choosing a {@link SentryClientFactory}.
+     *
+     * @return SentryClient
+     */
+    public static SentryClient init() {
+        return init(null, null);
+    }
+
+    /**
+     * Initialize and statically store a {@link SentryClient} by looking up
+     * a {@link Dsn} and using the provided {@link SentryClientFactory}.
+     *
+     * @param sentryClientFactory SentryClientFactory to use.
+     * @return SentryClient
+     */
+    public static SentryClient init(SentryClientFactory sentryClientFactory) {
+        return init(null, sentryClientFactory);
+    }
+
+    /**
+     * Initialize and statically store a {@link SentryClient} by using the provided
+     * {@link Dsn} and automatically choosing a {@link SentryClientFactory}.
+     *
+     * @param dsn Data Source Name of the Sentry server.
+     * @return SentryClient
+     */
+    public static SentryClient init(String dsn) {
+        return init(dsn, null);
+    }
+
+    /**
+     * Initialize and statically store a {@link SentryClient} by using the provided
+     * {@link Dsn} and {@link SentryClientFactory}.
+     * <p>
+     * Note that the Dsn or SentryClientFactory may be null, at which a best effort attempt
+     * is made to look up or choose the best value(s).
+     *
+     * @param dsn                 Data Source Name of the Sentry server.
+     * @param sentryClientFactory SentryClientFactory to use.
+     * @return SentryClient
+     */
+    public static SentryClient init(String dsn, SentryClientFactory sentryClientFactory) {
```

Comment:
These new `init` methods are the main focus here. The idea is that people who aren't using logging integrations will only want  to use `Sentry.init`, `Sentry.record(breadcrumb)`, `Sentry.capture(error)` etc.